### ISSUE TITLE
Rails 7.2 support

### ISSUE THREAD

**Issue by Piioo** [cockroachdb/activerecord-cockroachdb-adapter#342]
Hello,

is there a roadmap to add support for Rails 7.2 soon?

**Comment by BuonOmo** [comment]
Yes, latest commmit on the master branch! I guess @fqazi, @annrpom, or @rafiss should soon push it to rubygems

**Comment by BuonOmo** [comment]
It has been pushed now!
